app/models/katello/glue/pulp/repo.rb

```diff
@@ -176,21 +192,19 @@ def generate_distributors
           puppet_install_dist = Runcible::Models::PuppetInstallDistributor.new(repo_path, dist_options)
 
           dist_options[:id] = "#{self.pulp_id}_puppet"
-          puppet_dist = Runcible::Models::PuppetDistributor.new(self.relative_path, (self.unprotected || false),
+          puppet_dist = Runcible::Models::PuppetDistributor.new(nil, (self.unprotected || false),
```

Comment:
actually, ignore that, I didn't add it, but I guess it was there because unprotected wasn't a method on the other repo types, I think I can remove it without issue